.github/workflows/functions.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Copyright 2017-2020 O2 Czech Republic, a.s.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+proxima_version() {
+  oldPwd=$(pwd)
+  cd "${oldPwd}/$(dirname $BASH_SOURCE)/../../"
+  local version=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -x "^[^\[]*")
+  cd $oldPwd
+  if [ -z "${version}" ]; then
+    messageFail "Unable to get version"
+    RET=1
+  else
+    echo $version
+  fi
+}
+
+is_snapshot_version() {
+  [ -z "$1" ] && messageFail "Missing argument 1" && RET=1
+
+  echo $1 | grep -i "SNAPSHOT" > /dev/null
+  if [ $? -eq 0 ]; then
+    echo 1
+  else
+   echo 0
+  fi
+}
+
+is_datadriven_repo() {
```

Comment:
Can we reuse this call in the push to o2 action?

Reply:
Changed + added prevention to overwrite ~/.ssh/id_rsa* 🤦‍♂